.github/workflows/macos.yml

```diff
@@ -0,0 +1,431 @@
+# Copyright 2025 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+name: KTX-Software Linux CI
```

Comment:
name and filename don't match

Reply:
Yes, the PR is draft, not intended for review yet (although some things can be fixed now). (GA remembers workflow names, so I didn't want to give it Linux/MacOS becuase ultimately I wanted to have the platforms be separate workflows, but the 1st crude port behaves much like the Travis version. It's in an interim state and the final version will be clean from this perspective.)

Reply:
Understood. Thanks.